doc/click-to-debug.md

```diff
@@ -0,0 +1,81 @@
+# `@uu-cdh/backbone-util/src/click-to-debug.js`
+
+[view source](https://github.com/CentreForDigitalHumanities/backbone-util/tree/main/src/click-to-debug.js)
+[package README](../README.md)
+
+> This module depends on jQuery, which is marked as an `optionalDependency` of `@uu-cdh/backbone-util`.
+
+This module enables you to debug your live application by simply clicking on any view while keeping the alt (option, meta) key pressed. This logs the complete contents of the view to the developer console, so you can inspect its `model`, `collection`, `el`, `cid`, subviews, event listeners, etcetera. Ancestor views (i.e., views of which the `.el` envelops the `.el` of the clicked view) are logged as well. Three ingredients are needed to unlock this magic:
+
+1. Obtain our [mixin](#alt-click-view-mixin) by calling the [`getAltClickMixin` function](#function-getaltclickmixin).
+2. Define a common base class for all your views, which [has the mixin added in](using-mixins.md) and which calls `this.enableAltClick()` in its `initialize` method or `constructor`.
+3. Set `window.DEBUGGING = true` at the start of your application (or omit this, if you want to disable the behavior in production).
```

Comment:
Wouldn't it be more stable to use environment variables instead?

Reply:
Please clarify, what do you mean by an environment variable in this case?